processing/src/main/java/io/druid/segment/incremental/IncrementalIndex.java

```diff
@@ -372,6 +374,16 @@ public Builder setSimpleTestingIndexSchema(final AggregatorFactory... metrics)
       return this;
     }
 
+    @VisibleForTesting
+    public Builder setSimpleTestingIndexSchema(boolean rollup, final AggregatorFactory... metrics)
```

Comment:
Could you modify the other `setSimpleTestingIndexSchema` to call this one, and add a javadoc to this one?

Reply:
Sure, though I think I need to change the new method parameter to be `@Nullable Boolean rollup` and check if null to not set `.withRollup` (or not) to preserve the original's intention:

>A helper method to set a simple index schema _with only metrics and default values_ for the other parameters.